internal/providers/terraform/azure/frontdoor.go

```diff
@@ -0,0 +1,40 @@
+package azure
+
+import (
+	"strings"
+
+	"github.com/infracost/infracost/internal/resources/azure"
+	"github.com/infracost/infracost/internal/schema"
+)
+
+// getAzureRMFrontdoorRegistryItem returns a registry item for the resource
+func getAzureRMFrontdoorRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "azurerm_frontdoor",
+		RFunc: newFrontdoor,
+		ReferenceAttributes: []string{
+			"resource_group_name",
+		},
+	}
+}
+
+// newFrontdoor parses Terraform's data and uses it to build a new resource
+func newFrontdoor(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	region := lookupRegion(d, []string{"resource_group_name"})
+
+	if strings.HasPrefix(strings.ToLower(region), "usgov") {
+		region = "US Gov Zone 1"
+	} else {
+		region = regionToZone(region)
+	}
+
+	r := &azure.Frontdoor{
+		Address:               d.Address,
+		Region:                region,
+		FrontendHostsQuantity: len(d.Get("frontend_endpoint").Array()),
+		RoutingRulesQuantity:  len(d.Get("routing_rule").Array()),
```

Comment:
As per previous comment, I think we should only count enabled `routing_rule` blocks for this resource?